receiver/splunkhecreceiver/receiver.go

```diff
@@ -58,11 +58,11 @@ var (
 	errInvalidMethod          = errors.New("invalid http method")
 	errInvalidEncoding        = errors.New("invalid encoding")
 
-	okRespBody                = initJSONResponse(responseOK)
-	eventRequiredRespBody     = initJSONResponse(responseErrEventRequired)
-	eventBlankRespBody        = initJSONResponse(responseErrEventBlank)
+	okRespBody                = []byte(responseOK)
+	eventRequiredRespBody     = []byte(responseErrEventRequired)
+	eventBlankRespBody        = []byte(responseErrEventBlank)
 	invalidEncodingRespBody   = initJSONResponse(responseInvalidEncoding)
-	invalidFormatRespBody     = initJSONResponse(responseInvalidDataFormat)
+	invalidFormatRespBody     = []byte(responseInvalidDataFormat)
 	invalidMethodRespBody     = initJSONResponse(responseInvalidMethod)
 	errGzipReaderRespBody     = initJSONResponse(responseErrGzipReader)
 	errUnmarshalBodyRespBody  = initJSONResponse(responseErrUnmarshalBody)
```

Comment:
Can we update all the strings to the json format and remove `initJSONResponse` to be consistent?

Reply:
I guess. This has been ugly forever, true.

Reply:
Done!